task16/index.js

```diff
@@ -268,11 +90,54 @@ const createArticleTitle = ({ article }) => {
     articleTitle.textContent = article[i].title;
     articleLink.href = "#";
 
-    articleTitleFragment.appendChild(articleTitleItem).appendChild(articleWrapper).appendChild(articleLink).appendChild(articleTitle);
+    articleTitleFragment
+      .appendChild(articleTitleItem)
+      .appendChild(articleWrapper)
+      .appendChild(articleLink)
+      .appendChild(articleTitle);
+
+    const articleDate = elapsedTime(article[i].date);
+    const newArrivalsLimiter = 3;
+    if (articleDate <= newArrivalsLimiter) {
+      articleLink.appendChild(createNewArrivals());
+    }
+
+    const commentLength = article[i].comment.length;
+    if (commentLength > 0) {
+      articleLink.appendChild(createCommentCount(article[i]));
+    }
```

Comment:
IMO
It looks like the process here can be cut out.
Please think about the function name.
[codesandbox](https://codesandbox.io/s/tender-river-wq1slk?file=/index.js)
```js
   hogeDate(article[i], articleLink);
   hogeComment(article[i], articleLink);
```

```js
const hogeDate = (article, articleLink) => {
  const articleDate = elapsedTime(article.date);
  const newArrivalsLimiter = 3;
  if (articleDate <= newArrivalsLimiter) {
    articleLink.appendChild(createNewArrivals());
  }
};

const hogeComment = (article, articleLink) => {
  const commentLength = article.comment.length;
  if (commentLength > 0) {
    articleLink.appendChild(createCommentCount(article));
  }
};
```

Reply:
Thank you for pointing this out. I'm glad I did, because that's where I wanted to subdivide, but couldn't! I learned a lot.
4da4b29